### PR TITLE
LPS-167868 Create unchecked action for selected subtasks

### PR DESCRIPTION
**Pull request by rafaelfs98**

### CODE REVIEW

**Comment by rafaelfs98** [comment]
ci:test:sf

**Comment by rafaelfs98** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/FloatingBox/FloatingBox.tsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+import i18n from '../../i18n';
+
+type ButtonProps = React.ButtonHTMLAttributes<HTMLButtonElement> & {
+	loading?: boolean;
+};
+
+type FloatingBoxProps = {
+	alerts?: {text: string; title?: string}[];
+	clearList: () => void;
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/FloatingBox/FloatingBox.tsx:28]: Can you rename to `onClearList` ?

clearList doesn't looks like an action

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/FloatingBox/FloatingBox.tsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+	isVisible: boolean;
+	onSubmit?: () => void;
+	primaryButtonProps?: ButtonProps;
+	selectdCount?: number;
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/components/FloatingBox/FloatingBox.tsx:32]: This name is still wrong

**Review by kevenleone — CHANGES_REQUESTED** [review]

**Review by kevenleone — CHANGES_REQUESTED** [review]
@rafaelfs98 I'm missing the suggestions the I left for you here: https://github.com/pt-liferay-solutions/liferay-portal/pull/2635

Also, some of suggestions is not in this PR, can you take a look?

**Pull request closed without merging**